packages/react/src/components/RadioGroup/index.tsx

```diff
@@ -157,22 +165,23 @@ RadioGroup.propTypes = {
       id: PropTypes.string.isRequired,
       label: PropTypes.string.isRequired,
       labelDescription: PropTypes.string
-    })
+    }).isRequired
   ).isRequired,
   hasLabel: (
     props: { [key: string]: string },
-    propName: string,
+    _propName: string,
     componentName: string
-  ) => {
+  ): null | Error => {
     if (!props['aria-label'] && !props['aria-labelledby']) {
       return new Error(
         `${componentName} must have an "aria-label" or "aria-labelledby" prop`
       );
     }
+    return null;
   },
   className: PropTypes.string,
   defaultValue: PropTypes.string,
-  onChange: PropTypes.func
```

Comment:
This shouldn't be required because we do set a default.

Reply:
Since we pass a default, shouldn't we make the [TypeScript type here optional as well](https://github.com/dequelabs/cauldron/blob/82aade9999dc85bc103c2db51539c645fccc8969/packages/react/src/components/RadioGroup/index.tsx#L19). Otherwise, my linter is complaining on this line:

```
Type 'Requireable<(...args: any[]) => any>' is not assignable to type 'Validator<(radio: RadioItem, input: HTMLElement) => void>'.
  Types of property '[nominalTypeHack]' are incompatible.
    Type '{ type: ((...args: any[]) => any) | null | undefined; } | undefined' is not assignable to type '{ type: (radio: RadioItem, input: HTMLElement) => void; } | undefined'.
      Type '{ type: ((...args: any[]) => any) | null | undefined; }' is not assignable to type '{ type: (radio: RadioItem, input: HTMLElement) => void; }'.
        Types of property 'type' are incompatible.
          Type '((...args: any[]) => any) | null | undefined' is not assignable to type '(radio: RadioItem, input: HTMLElement) => void'.
            Type 'undefined' is not assignable to type '(radio: RadioItem, input: HTMLElement) => void'.ts(2322)
```